aboutUs.html

```diff
@@ -1,6 +1,71 @@
 <DOCTYPE! <!DOCTYPE html>
     <html>
 
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Page Title</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" type="text/css" media="screen" href="style.css">
+        <script src="main.js"></script>
+    </head>
+
+    <body>
+        <div id="container">
+            <div class="grid-container">
+                <div>
+                    <img src="Thomas'Pics/LittleThomas.PNG" alt="AndyWang" width="500" height="500">
+                </div>
+                <div class="ThomasText">
+                    This is Thomas
+                </div>
+            </div>
+            <div class="grid-container">
+                <div class="ThomasText">
+                    Thomas is also a king
+                </div>
+                <div>
+                    <img src="Thomas'Pics/AcneThomas.PNG" width="500" height="500">
+                </div>
+            </div>
+            <div class="grid-container">
+                <div>
+                    <img src="Thomas'Pics/ThiccThomas.PNG" alt="AndyWang" width="500" height="500">
+                </div>
+                <div class="ThomasText">
+                    Thomas is also an NBA player
+                </div>
+            </div>
+            <div class="grid-container">
+                <div class="ThomasText">
+                    Thomas is also considered very attractive
+                </div>
+                <div>
+                    <img src="Thomas'Pics/TurtleThomas.PNG" width="500" height="500">
+                </div>
+            </div>
+            <div class="grid-container">
+                <div>
+                    <img src="Thomas'Pics/boollingThomas.PNG" alt="AndyWang" width="500" height="500">
+                </div>
+                <div class="ThomasText">
+                    Thomas is also a hacker.
+                </div>
+            </div>
+            <div class="grid-container">
+                <div class="ThomasText">
+                    Thomas is also a member of the Andy Wang church.
+                </div>
+                <div>
+                    <img src="Thomas'Pics/AndyWang.jpg" width="500" height="500">
+                </div>
+            </div>
+        </div>
+    </body>
+
+    </html>
+    <html>
+
     <head>
```

Comment:
You left two headers, this will break the site. Mine has the bootstrap in it so switch yourh head ot my head